OpenSim/Common/C3DFileAdapter.h

```diff
@@ -44,10 +42,13 @@ class OSIMCOMMON_API C3DFileAdapter : public FileAdapter {
     C3DFileAdapter& operator=(C3DFileAdapter&&)      = default;
     ~C3DFileAdapter()                                = default;
 
+    explicit C3DFileAdapter(const bool& useSIMMColumnOrdering);
```

Comment:
Adding the logic of reordering columns to STOFileAdapter::write() :
* Assumes that the table being written contains forces.
* Requires to logic to find the columns by their label (and not index like the current solution) for reordering. This will need the columns to be labeled in with suffixes assumed by the function or suffixes will have to be provided to the function as arguments. This will also need more error-checking to be correct -- for example, some columns may be missing, or not labeled correctly.